rpm/harbour-webpirate.yaml

```diff
@@ -3,9 +3,9 @@ Summary: harbour-webpirate
 Version: 1.6
 Release: 3
 # The contents of the Group field should be one of the groups listed here:
-# http://gitorious.org/meego-developer-tools/spectacle/blobs/master/data/GROUPS
+# https://gitorious.org/meego-developer-tools/spectacle/blobs/master/data/GROUPS
```

Comment:
yaml file is autogenerated by QtCreator, I've never noticed this line, it's only a comment, however it can be changed, doesn't affect anything